images/linux/scripts/installers/java-tools.sh

```diff
@@ -79,13 +88,15 @@ done
 
 # Document what was added to the image
 echo "Lastly, documenting what we added to the metadata file"
-DocumentInstalledItem "Azul Zulu OpenJDK:"
+DocumentInstalledItem "Adopt OpenJDK:"
```

Comment:
I believe JDK7 is still Azul Zulu, so we need to mention it somehow.

Reply:
I agree with you. Thank you.